randovania/games/super_metroid/json_data/Norfair.txt

```diff
@@ -1830,7 +1833,9 @@ Extra - asset_id: 251
   * Layers: default
   * Normal Door to Main Hall/Door to Acid Statue Room
   > Door to Golden Torizo's Room
-      Morph Ball and Power Bombs and Space Jump and Varia Suit
+      All of the following:
+          Varia Suit and Can Lay Power Bombs
+          Space Jump or Enabled No Lower Norfair Space Jump Check
 
```

Comment:
OR
- Spacejump
- And
    - No Lower Norfair Space Jump Check
    - A way to get out of the acid
Could be gravity suit and HJ. Gravity and WJ beginner. Gravity and Springball jump. Gravity and IBJ... probably some more I can't think of right now. I'll have to look at it more closely later.

Reply:
I completely forgot that there was acid in there, whoops.